src/main/java/com/google/sps/servlets/DBUtilities.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import com.google.sps.servlets.Property;
+
+public class DBUtilities {
+
+  public static final String INTERACTION_TABLE = "Interactions";
+
+ /**
+  * Given an HTTP request object, this method will get the correlator parameter
+  * and return it. Returns an empty string if correlator is not given.
+  */
+  public static String getCorrelator(HttpServletRequest request) {
```

Comment:
This doesn't seem like it belongs in a class called DBUtils, since it has nothing to do with the database

Reply:
Where would you suggest I put this? I wasn't a fan of putting it in a separate class with just one method but I also want it to be shared.

Reply:
Putting it in its own class is fine (RequestUtils?), that way if you end up adding more functions you already have a place to put them.

Alternatively, you could rename this class to just utils, but imo itd be better to keep them separate

Reply:
Okay, done!